css-transitions-2/Overview.bs

```diff
@@ -72,12 +72,29 @@ spec:web-animations-1; type:dfn;
 
 ## The 'transition-property' Property ## {#transition-name-property}
 
+ISSUE: Insert text from [[css-transitions-1#transition-property-property]]
+and backport the following paragraph.
+
 Although the 'transition-property' may specify shorthand properties and the
 ''all'' keyword, individual transitions are generated for each longhand
 sub-property that is animatable. The <dfn>expanded transition property
 name</dfn> of a transition is the name of the longhand sub-property for which
 the transition was generated (e.g. 'border-left-width').
 
+ISSUE: Update the defining instance of [=transitionable=]
+once it is ported from Level 1.
+
+In CSS Transitions Level 2,
+property values are [=transitionable=]
+<em>unless</em> they have an [=animation type=] that is [=not animatable=],
+i.e. [=discrete=] values <em>are</em> [=transitionable=],
+and flip at 50% progress.
+However, because authors will rarely want [=discrete=] values to transition at 50%
+(and for compatibility with CSS Transitions Level 1)
+the ''transition-property/all'' keyword
+expands only to all [=transitionable=] properties
```

Comment:
I think this should probably be its own paragraph (since it's specifically about changing the definition of `all`), and perhaps should be more clearly split into (1) a change to the description of `all` and (2) the motivation for that change.

Reply:
Done.